MPF.Core/Utilities/Tools.cs

```diff
@@ -71,6 +72,35 @@ private static bool EqualAt(this byte[] stack, byte[] needle, int index)
             return true;
         }
 
+        /// <summary>
+        /// Converts a hex string into a byte array
+        /// </summary>
+        /// <param name="hex">Hex string</param>
+        /// <returns>Converted byte array, or null if invalid hex string</returns>
+        public static byte[]? HexStringToByteArray(string? hexString)
```

Comment:
I think there's a helper method for this already?

Reply:
Not that I have seen, this is the same function of mine I originally had in CreateIRDViewModel.
There *is* a slower in-built function for this, but its not available for earlier .NET versions.